documentation/src/main/asciidoc/topics/proc_server_configuring_ssl_engine.adoc

```diff
@@ -25,7 +25,7 @@ Omitting the `enabled-protocols` attribute allows any TLS version.
 ----
 ====
 +
-. Configure {brandname} to use one or more cipher suites with the `enabled-ciphersuites` attribute.
+. Configure {brandname} to use one or more cipher suites with the `enabled-ciphersuites-filter` attribute (for TLSv1.2 and below) and the `enabled-ciphersuites-names` (for TLSv1.3).
```

Comment:
I'm no TLS expert, but are the terms "filter" and "names" synonymous with `TLS < 1.3` and `1.3` respectively?

To me keeping `enabled-ciphersuites` for v1.1 and v1.2 and adding `enabled-ciphersuites-1-3` for v1.3 is much more readable and self-explanatory.

Reply:
I went with the WildFly names. I looked at the OpenSSL wiki https://wiki.openssl.org/index.php/TLS1.3 and they use "ciphers" for specifying filters for <= 1.2 and "ciphersuites" for 1.3.
The difference is not just in the protocols they apply to, but in the way they affect the available ciphersuites. The filter allows selecting whole familiies. The names select specific ciphersuites. And unfortunately there is no overlap between the two.
I like your proposal, but I don't like hardcoding the version in the attribute name...